Documentation/network/node-ipam.rst

```diff
@@ -15,11 +15,33 @@ to "advertise" the node's IPs directly inside a Service LoadBalancer. This featu
 is especially useful if you don't control the network you are running on and can't
 use either the L2 or BGP capabilities of Cilium.
 
-It works by getting the Node addresses of the selected pods and advertising them.
+It works by getting the Node addresses of the selected Nodes and advertising them.
 It will respect the ``.spec.ipFamilies`` to decide if IPv4 or IPv6 addresses
 shall be used and will use the ``ExternalIP`` addresses if any or the
 ``InternalIP`` addresses otherwise.
 
+If the Service has ``.spec.externalTrafficPolicy`` sets to ``Cluster`` Node IPAM
+will consider all your nodes as candidate for selection. Or else if
+``.spec.externalTrafficPolicy`` is set to ``Local``, Node IPAM will consider
+all the Pods selected by the Service (via their EndpointSlices) as candidate.
```

Comment:
```suggestion
If the Service has ``.spec.externalTrafficPolicy`` set to ``Cluster``, Node IPAM
considers all nodes as candidates for selection. Otherwise, if
``.spec.externalTrafficPolicy`` is set to ``Local``, then Node IPAM considers
all the Pods selected by the Service (via their EndpointSlices) as candidates.
```